lib/dish.rb

```diff
@@ -0,0 +1,9 @@
+class Dish
+
+  attr_reader :name, :category
+
+  def initialize(name, category)
+    @name = name
+    @category = {category => name}
```

Comment:
Not sure why this is here. From the spec, category should be the exact same value passed in as a parameter